### ISSUE TITLE
FR: Timeline Plugin

### ISSUE THREAD

**Issue by rejochandran** [dschoon/react-waves#9]
Hi @dschoon,

Good day!

The timeline plugin will be a great add. 
Working on the fork. Let know if you have any updates on it.

Thank you and wishing best! 🙂

**Comment by dschoon** [comment]
Hey @rejochandran, thanks for opening this!

I'd like to add all of the available plugins eventually. I'll try my best to add the Timeline one next 🙂

**Comment by harulkanahasoft** [comment]
sddfdsfsdaf

**Comment by rejochandran** [comment]
Hey @dschoon,

Please let know if you got any update on this. Eager to get this on 🤩

In the meantime, is it possible to use `Timeline` with the `plugins` attribute ?
something like this,
```
<ReactWaves
...
    options={{
        plugins:[
			Timeline.create({
			       container:'#timeline-container'
			})		
		]
    }}
...
</ReactWaves>
```

Thank you

**Comment by dschoon** [comment]
Hey @rejochandran, thanks for submitting this feature request. I'm very sorry that it took me so long to get around to implementing it. This capability is now included in the new major version (v3.0.0).

**Demo:** https://dschoon.github.io/react-waves/

**Example Code:** https://github.com/dschoon/react-waves/blob/master/example/src/components/TimelineExample.js
